go/filemanager/pfsmodules/localfile.go

```diff
@@ -0,0 +1,121 @@
+package pfsmodules
+
+import (
+	"bytes"
+	"crypto/md5"
+	"encoding/hex"
+	"io"
+	"os"
+
+	log "github.com/golang/glog"
+)
+
+// FileHandle is a local *os.File with offset.
+type FileHandle struct {
+	F      *os.File
+	Offset int64
+}
+
+// Close closes FileHandle.
+func (f *FileHandle) Close() {
+	if f.F != nil {
+		f.F.Close()
+	}
+}
+
+// Open opens a file.
+func (f *FileHandle) Open(path string, flag int, size int64) error {
+	if flag == WriteOnly ||
+		flag == ReadAndWrite {
+
+		cmd := TouchCmd{
```

Comment:
`os.OpenFile` in few lines below already "touch" the file, why we still need to do "touch" here?